content/golanguk-conference.md

```diff
@@ -0,0 +1,32 @@
++++
+author = ["Alexandre González"]
+date = "2015-03-27T20:30:00-00:00"
+linktitle = "The Golang UK Conference"
```

Comment:
Hi, I'm one of organizers too. The name of the conference is `Golang Uk Conference` and the link is `http://www.golanguk.com/` we know that is not the name of the language, but is convenience for us, so the link make sense to reference the current name of the conference.

And to be honest, personally I don't care to much about the name, I hope it could be a good conference where the people enjoy Go